neo4j/config.go

```diff
@@ -0,0 +1,102 @@
+package neo4jstore
```

Comment:
@the-real-i9 I was looking at the other drivers and this can be named `neo4j`. For example our redis one is named `package redis` same as the upstream one and it doesn't collide.